addons/purchase_stock/models/purchase.py

```diff
@@ -316,6 +317,16 @@ def _compute_qty_received(self):
                 line._track_qty_received(total)
                 line.qty_received = total
 
+    @api.depends('qty_invoiced', 'date_planned')
+    def _compute_forecasted_issue(self):
```

Comment:
Same than the other comment. If the PO is draft, shouldn't take in account the quantity of the current purchase line ?